modules/openapi-generator/src/main/java/org/openapitools/codegen/languages/PythonExperimentalClientCodegen.java

```diff
@@ -1687,13 +1687,15 @@ private MappedModel getDiscriminatorMappedModel(CodegenDiscriminator disc) {
      */
     private void addSchemaToIncludedSchemaList(List<Schema> includedSchemas, Schema schema) {
 
-        // if the schema is in the list, then throw an exception.
-        // this should never happen
         if(includedSchemas.contains(schema)) {
-            throw new RuntimeException("Attempted to add a schema to the includedSchemas list twice");
+            String title = schema.getTitle();
+            if(title == null && schema instanceof ArraySchema) {
+                title = "Array<" + ((ArraySchema) schema).getItems().get$ref() + ">";
+            }
+            LOGGER.warn("Attempted to add a schema " + title + " to the includedSchemas list twice");
```

Comment:
How about getting rid of this function and every place you invoke it if the schema you are adding is seen, then return empty string.